.github/actions/promote_docker_image/action.yml

```diff
@@ -0,0 +1,65 @@
+name: Promotes built Docker images to target environment registries for consumption
+
+inputs:
+  project_name:
```

Comment:
I would probably define the types for each input

Reply:
Afaik these actions only support one type, strings, but I can add them.

Reply:
Ahh that's right, carry on ;)